tools/pythonpkg/tests/arrow/tpch/queries/q01.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
```

Comment:
Same for the queries, could we not export those from the tpch extension to Python, too?